arm-sql/2015-05-01/swagger/sql.json

```diff
@@ -0,0 +1,214 @@
+        "version": "2015-05-01-preview"
+    },
+    "host": "management.azure.com",
+    "schemes": [
+        "https"
+    ],
+    "consumes": [
+        "application/json",
+        "text/json"
+    ],
+    "produces": [
+        "application/json",
+        "text/json"
+    ],
+    "paths": {
+        "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.Sql/servers/{serverName}/securityAlertPolicies/Default": {
+            "put": {
+                "x-ms-long-running-operation": true,
+                "tags": [
+                    "SecurityAlertPolicy"
+                ],
+                "operationId": "SecurityAlertPolicy_CreateOrUpdateServerPolicy",
+                "description": "Creates or updates an Azure SQL Server security alert policy.",
+                "parameters": [
+                    {
+                        "name": "parameters",
+                        "in": "body",
+                        "required": true,
+                        "schema": {
+                            "$ref": "#/definitions/ServerSecurityAlertPolicyCreateOrUpdateParameters"
+                        },
+                        "description": "The required parameters for creating or updating a Azure SQL Server security alert policy."
+                    },
+                    {
+                      "name": "resourceGroupName",
+                      "in": "path",
+                      "required": true,
+                      "type": "string",
+                      "description": "The name of the Resource Group to which the server belongs."
+                    },
+                    {
+                      "name": "serverName",
+                      "in": "path",
+                      "required": true,
+                      "type": "string",
+                      "description": "The name of the Azure SQL Server."
+                    }, 
+                    {
+                        "$ref": "#/parameters/ApiVersionParameter"
+                    },
+                    {
+                        "$ref": "#/parameters/SubscriptionIdParameter"
+                    }
+                ],
+                "responses": {
+                    "200": {
+                        "description": "The Server Threat Detection Policy",
+                        "schema": {
+                            "$ref": "#/definitions/ServerSecurityAlertPolicy"
+                        }
+                    },
+                    "201": {
+                        "description": "The Server Threat Detection Policy",
+                        "schema": {
+                            "$ref": "#/definitions/ServerSecurityAlertPolicy"
+                        }
+                    }
+                }
+            },
+            "get": {
+                "tags": [
+                    "SecurityAlertPolicy"
+                ],
+                "operationId": "SecurityAlertPolicy_GetServerPolicy",
```

Comment:
I would recommend to update the naming for all the operations as described in the previous naming comment